### ISSUE TITLE
MaxMix classified as virus on PC

### ISSUE THREAD

**Issue by TheFoxBurrow** [t3knomanzer/maxmix-software#234]
**Describe the bug**
MaxMix has been removed by Windows Virus & threat protection and consequently been deleted. Redownload is also not possible as it immediately gets classified as virus and deleted.
**To Reproduce**
Well I guess try to download MaxMix without completely disabling your vanilla Windows security options.

**Expected behavior**
I exptected it not to be classified as a virus

**Screenshots**
Can add later if necessary

**System information:**
 - OS version: Windows 10 Professional Version 1909 (Build 18363.1139)
 - Firmware version: MaxMix device says 1.4.7 built on Nov 2 2020

**Comment by pbanj** [comment]
https://github.com/t3knomanzer/maxmix-software/issues/213

**Comment by TheFoxBurrow** [comment]
Yes, but now it's not just third party programs but also the native Windows one

**Comment by pbanj** [comment]
it doesnt change what was said.

**Comment by TheFoxBurrow** [comment]
Appreciate it, thank you

**Comment by pbanj** [comment]
it's annoying but microsofts response tells me they dont really give a damn. all we can do is submit the files to the anti virus programs and hope they remove the flag.
